docs/pages/admin-guides/deploy-a-cluster/helm-deployments/ibm.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: Running Teleport on IBM Cloud
+description: How to install and configure Teleport on IBM cloud
+---
+
+This guide describes how to set up Teleport on IBM Cloud.
+
+We will use the following services to deploy Teleport on IBM cloud:
+- [Compute: IBM Cloud Kubernetes Service](https://www.ibm.com/products/kubernetes-service)
+  to deploy Teleport using the [`teleport-cluster` Helm chart](../../../reference/helm-reference/teleport-cluster.mdx)
+- [Storage: IBM Cloud Databases for PostgreSQL](https://www.ibm.com/products/databases-for-postgresql)
+  to store the Teleport cluster state, and the Teleport audit events (see []backends reference](../../../reference/backends.mdx)
+  for more details).
+- [Storage: IBM Cloud Object Storage](https://www.ibm.com/products/cloud-object-storage) to store the Teleport
+  session recordings.
+- [Network: IBM Cloud DNS Services](https://www.ibm.com/products/dns) to route Teleport
+
+<Admonition type="note">
+A previous version of this guide covered how to deploy Teleport using VMs and etcd.
+While this deployment method still works, we now recommend to use the Postgres backend.
+Postgres offers better scalability, is easier to operate, and supports both Teleport
+core backend and audit events.
+</Admonition>
```

Comment:
Is this note really necessary? It refers to the previous guide but the previous guide is deleted and users can't find it unless they maybe explore older doc versions. I think it's ok to not mention it, I would remove this note.
```suggestion
```